book/graphics.md

```diff
@@ -607,8 +609,8 @@ these at the end of the chapter so you can see how it improves.]
 
 :::
 
-Next, we'll make this browser work on English text, with all its
-complexities of variable-width characters, line layout, and
+Next, we'll make this browser work on English text, handling
+complexities like variable-width characters, line layout, and
```

Comment:
I could go either way here but I think the replacement sounds like enumerating the complexities we handle, while the original sounds like enumerating the complexities of English text in general.